jung-algorithms/src/test/java/edu/uci/ics/jung/algorithms/generators/random/TestErdosRenyi.java

```diff
@@ -34,7 +35,7 @@ public void test() {
     int total = 0;
     for (int i = 1; i <= 10; i++) {
       ErdosRenyiGenerator<String> generator = new ErdosRenyiGenerator<>(nodeFactory, numNodes, 0.1);
-      generator.setSeed(0);
+      generator.setRandom(new Random());
```

Comment:
`Random()` doesn't guarantee anything about the seed to be used: https://docs.oracle.com/javase/8/docs/api/java/util/Random.html#Random--

The existing code was setting the random seed to a constant (0); we should preserve that behavior.

Reply:
Ooops. That was not by intention.

Reply:
No worries.  Thanks for the fix.  :)